Nodejs/Product/ProjectWizard/CloudServiceWizard.cs

```diff
@@ -37,6 +37,8 @@ public sealed class CloudServiceWizard : IWizard {
         const string AzureToolsDownload = "https://go.microsoft.com/fwlink/p/?linkid=323510";
 #elif DEV14
         const string AzureToolsDownload = "http://go.microsoft.com/fwlink/?LinkID=517353";
+#elif DEV15
+        const string AzureToolsDownload = "about:blank";
```

Comment:
Please add a todo in code, plus open a bug to track changing this before release.